packages/internals/src/get-generators/generatorResolvers/prisma-client-js/prismaClientResolver.ts

```diff
@@ -66,18 +68,18 @@ export async function prismaClientResolver(baseDir: string, version?: string) {
         `Could not resolve ${missingCli(`${bold('prisma')} and `)}${bold(
           '@prisma/client',
         )} in the current project. Please install ${hasCli('it')}${missingCli('them')} with ${missingCli(
-          `${bold(green(`${getPackageCmd(baseDir, 'add', 'prisma', '-D')}`))} and `,
-        )}${bold(green(`${getPackageCmd(baseDir, 'add', '@prisma/client')}`))}, and rerun ${bold(
+          `${bold(green(`${await getPackageCmd(baseDir, 'add', 'prisma', '-D')}`))} and `,
+        )}${bold(green(`${await getPackageCmd(baseDir, 'add', '@prisma/client')}`))}, and rerun ${bold(
```

Comment:
These two lines used to print `[object Promise]` due to missing `await`s. I can extracted this to a separate PR for Prisma 4.16.

Reply:
I think we have that already from a user: https://github.com/prisma/prisma/pull/19259

Reply:
ah, nice, thanks!

Reply:
will do that tomorrow then